nmrfx-processor/src/test/java/org/nmrfx/processor/datasets/vendor/RS2DDataTest.java

```diff
@@ -0,0 +1,100 @@
+package org.nmrfx.processor.datasets.vendor;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.nmrfx.processor.datasets.DatasetCompare;
+import org.w3c.dom.Document;
+
+import javax.xml.transform.TransformerException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class RS2DDataTest {
+    public static String fidHome = "../../nmrfxp_tests/testfids/";
```

Comment:
I wanted to change this code a bit, but I don't have the test data to ensure that I don't introduce any regression. Where can I find it?

Reply:
I will upload it and post the location :)